### PR TITLE
Fix bad display of scrollbar for sidebar menu in RTL

### LINKED ISSUE #27325: Bad display of scrollbar for sidebar menu in RTL
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

Scrollbar for sidebar menu in RTL is hidden on new-theme pages.

**RTL:**

**LTR:**

### Expected behavior

It must be displayed at the left side of sidebar-menu.

### Steps to reproduce

1. Go to BO > Orders (Open submenu)
2. Try to wheel inside menu
3. Scrollbar will be displayed at right side with an incorrect width. 

### PrestaShop version(s) where the bug happened

8.0

### PHP version(s) where the bug happened

8.0

### If your bug is related to a module, specify its name and its version

_No response_

### PR DESCRIPTION
**Pull request by mparvazi**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | Fix scrollbar position in sidebar menu in BO for RTL version.
| Type?             | bug fix
| Category?         | BO 
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #27325
| How to test?      | Go to BO > Orders
| Possible impacts? | Sidebar menu UI in RTL version (new theme)

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/27333)

### CODE REVIEW

File: admin-dev/themes/new-theme/public/theme.rtlfix [hunk 1]
```diff
@@ -132,3 +132,8 @@
   border-right-style: none;
   border-left-style: solid;
 }
+
+/* Fix scrollbar position in sidebar menu */
```
Comment (PululuK) [inline admin-dev/themes/new-theme/public/theme.rtlfix:136]:
```suggestion
```

**Review by PululuK — APPROVED** [review]
We can remove this comment I think

**Review by saulaski — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

File: admin-dev/themes/new-theme/public/theme.rtlfix (hunk 1 above)
Comment (NeOMakinG) [reply admin-dev/themes/new-theme/public/theme.rtlfix:136]: I would let it, not that bad because it's not that understandable

**Comment by marwachelly** [comment]
Hello,

Thanks @mparvazi  for you PR. It is Ok :+1:  

https://watch.screencastify.com/v/OfJeyglPkx2AOTaNflk3

Thanks,

**Review by Progi1984 — CHANGES_REQUESTED** [review]
WAIT FOR 1783 RELEASE

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @marwachelly & @mparvazi

**Pull request merged**